crates/adapters/dydx/tests/data.rs

```diff
@@ -14,3 +14,558 @@
 // -------------------------------------------------------------------------------------------------
 
 //! Integration tests for dYdX data client.
+//!
+//! These tests verify:
```

Comment:
I'm on the fence about comments like this. For a ~2000 line test module they can help to describe what is actually being tested (in a table format).

However, these comments can easily go stale, so are expensive to maintain, and the function names already describe this (so I think we should remove these in a follow-up).